docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -1177,29 +1176,28 @@ def __init__(self, dataloader_kNN, num_classes):
         self.projection_head = heads.BarlowTwinsProjectionHead(512, 2048, 2048)
         self.local_projection_head = heads.VicRegLLocalProjectionHead(512, 128, 128)
         self.average_pool = nn.AdaptiveAvgPool2d(output_size=(1, 1))
-        self.criterion = VICRegLLoss(alpha=0.75, num_matches=(16, 4))
+        self.criterion = VICRegLLoss(num_matches=(8, 4))
         self.backbone = nn.Sequential(self.train_backbone, self.average_pool)
-        self.warmup_epochs = 40 if max_epochs >= 800 else 20
+        self.warmup_epochs = 20 if max_epochs >= 800 else 10
```

Comment:
Reduced this because the paper uses only 10 epochs warmup for 300 epochs training.